solidity/dashboard/.git-blame-ignore-revs

```diff
@@ -0,0 +1,6 @@
+# This file contains commit hashes of bulk changes that can be omitted by the
```

Comment:
Can we put `.git-blame-ignore-revs` file in the repository root directory?

Reply:
please see e4c5ae3